src/component/books/books.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Form from '../home/form';
+import Showbooks from './Showbooks';
+
+const Books = () => {
+  const homeStyles = [
+    {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+  ];
+  return (
+    <div style={homeStyles[0]}>
+      <Showbooks />
+      <Showbooks />
+      <Showbooks />
```

Comment:
- [ ] In addition to upper comment, you can pass the title and author to this `Showbooks` component. 

Hint:- `<Showbooks title = {title} author={author} />`